code/src/App.js

```diff
@@ -1,12 +1,27 @@
 import React from 'react';
 import data from './data.json';
+import Card from './components/Card'
 
-console.log(data);
-
+// named export
 export const App = () => {
   return (
-    <div>
-      Find me in src/app.js!
-    </div>
+  <> 
+<section class="container">
+  <h1>New albums and singles</h1>
```

Comment:
This h1, according to the sketch, should have a bottom border. This can be achieved through adding a bottom border in CSS on h1 or maybe a div class of header that is styled :-)